docs/articles/advanced_topics/buttons.md

```diff
@@ -0,0 +1,117 @@
+---
+uid: advanced_topics_buttons
+title: Buttons & Components
+---
+
+# Button Introduction
+Buttons are a feature in Discord based on the interaction framework. Buttons are appended to the bottom of a message, and come in several colors.
+
+Before working with buttons, you will want to familarize yourself with the [Message Builder](xref:beyond_basics_messagebuilder). It, among other builders will be mentioned throughout this article, and function similarly.
+
+With buttons, you can have up to five buttons in a row, and up to five (5) rows of buttons. 
+Furthermore, buttons come in two types: regular, and link. Link buttons contain a Url field, and are always grey.
+
+# Buttons Continued
+
+> [!WARNING]
+> Custom Ids on buttons should be unique, as this is what's sent back when a user presses a button.
+>
+> Link buttons do **not** have a custom id and do **not** send interactions when pressed.
+
```

Comment:
This is very out of place. What custom ID are you talking about here? Why am I being yelled at?

Reply:
I can move it down if you'd like?

Reply:
Emzi recommended warning. It was initially a note.